ext/date/date_core.c

```diff
@@ -4505,6 +4505,9 @@ date_s__parse_internal(int argc, VALUE *argv, VALUE klass)
  * <b>Note</b>:
  * This method recognizes many forms in +string+,
  * but it is not a validator.
+ * For formats, see
+ * {Formats for Dates and Times}[https://docs.ruby-lang.org/en/master/strftime_formatting_rdoc.html].
```

Comment:
I think it would  be more helpful to be specific on what we want the reader to refer to:

```suggestion
 * {"Specialized Format Strings" in Formats for Dates and Times}[https://docs.ruby-lang.org/en/master/strftime_formatting_rdoc.html#label-Specialized+Format+Strings]
```

Reply:
I don't think ::_parse is specialized.

Reply:
I'm not exactly sure what heuristics `Date.parse` and `Date._parse` uses, but it seems like every "Specialized Format String" works. At any rate, as a reader, I'm confused as to what part of that page I should be reading to help me understand `Date.parse` and `Date._parse`.